others/serialization.cpp

```diff
@@ -0,0 +1,163 @@
+/**
+ * @file
+ * @brief A simple Serializer and Deserializer utility for fundamental data
+ * types and strings.
```

Comment:
you should add more details via a `@detail` tag and your name with `@author` tag